app/jobs/spotlight/default_thumbnail_job.rb

```diff
@@ -0,0 +1,14 @@
+module Spotlight
+  ###
+  # Calls the #set_default_thumbnail method
+  # on the object passed in and calls save
+  ###
+  class DefaultThumbnailJob < ActiveJob::Base
+    queue_as :default
+
+    def perform(thumbnailable)
+      thumbnailable.set_default_thumbnail
+      thumbnailable.save
```

Comment:
Explicitly calling `#save` makes sense to me. If it wasn't called, there's a good chance we'd forget somewhere along the line.